src/components/ChatEntry.js

```diff
@@ -1,22 +1,47 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './ChatEntry.css';
 import PropTypes from 'prop-types';
+import TimeStamp from './TimeStamp';
 
 const ChatEntry = (props) => {
+  const [liked, setLike] = useState(false);
+  const changeHeart = !liked ? '🤍' : '❤️';
+  const changeLike = (id) => {
+    const likeState = setLike(!liked);
+    props.onChangeLike(id);
+
+    return likeState;
+  };
+
   return (
     <div className="chat-entry local">
```

Comment:
Here we have the class `chat-entry local` hardcoded to this div so when I start up your react app, all the dialogue bubbles are on the left side. We need a way to dynamically change the class between `chat-entry local` and `chat-entry remote`.

How would you use a variable to reference the different classes depending on if the ChatEntry belongs to Vladimir or Estragon?

Consider checking to see if the value of sender is 'Vladimir', if it is then the value of the class should be `chat-entry local` otherwise it should be `chat-entry remote`. How would you use a ternary to do this?